recipes/snakeatac_env/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "7d9c753aceadd0cdd2132f510375e191f1bd9de92e29e43d710433ca770e1f03" %}
+
+package:
+  name: snakeatac_env
+  version: {{ version }}
+
+source:
+  - url: https://github.com/sebastian-gregoricchio/snakeATAC/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+build:
+  noarch: python
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('snakeatac_env', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy >=1.22
+    - conda-forge::mpich
```

Comment:
Why some dependencies are prefixed?

Reply:
Because certain steps in the [`snakeATAC`](https://github.com/sebastian-gregoricchio/snakeATAC) pipeline require functions that are available only in certain versions of the packages, or I know that in certain packages bugs that break the pipeline have been fixed from a specific version

Reply:
I actually had some issue during the local testing, that's why I added it.
I removed it now